app/views/api/v2/smart_proxies/import_subnets.json.rabl

```diff
@@ -0,0 +1,13 @@
+object @smart_proxy
+
+node :success do
+  locals[:success]
+end
```

Comment:
Why is `success` needed? The status code already indicates that so I'd leave it out. Note how there is no branch where `:success => false`.